std/src/main/scala/cats/std/map.scala

```diff
@@ -0,0 +1,38 @@
+package cats
+package std
+
+
+trait MapInstances {
+  type MapWithK[A] = Map[A, _]
```

Comment:
you can use kind projector to avoid type lamda, e.g. `Map[A, ?] = ({ type l[b] = Map[A, b] })#l `